Tests/AppTests/MarkdownHTMLConverterTests.swift

```diff
@@ -32,7 +32,7 @@ class MarkdownHTMLConverterTests: XCTestCase {
         try XCTSkipIf((Environment.get("SKIP_SNAPSHOTS") ?? "false") == "true")
         Current.date = { Date(timeIntervalSince1970: 0) }
         TempWebRoot.cleanup()
-        SnapshotTesting.isRecording = true
+        SnapshotTesting.isRecording = false
```

Comment:
Swear I told myself to change this before committing on about 5 different occasions and still forgot 😂  My bad!